src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/cas1/Cas1ApAreaMappingService.kt

```diff
@@ -47,9 +47,9 @@ class Cas1ApAreaMappingService(
     usersProbationRegion: ProbationRegionEntity,
     deliusUser: StaffUserDetails,
   ): ApAreaEntity {
-    val deliusProbationAreaCode = deliusUser.probationArea.code
-    val isInNationalProbationArea = listOf("N43", "N41", "XXX").contains(deliusProbationAreaCode)
-    if (isInNationalProbationArea) {
+    if (usersProbationRegion.apArea != null) {
```

Comment:
nice! -- thanks for spotting and using this opportunity to remove these magic probation area codes.